addons/l10n_th/data/account_data.xml

```diff
@@ -15,9 +15,6 @@
         <record id="tax_group_5" model="account.tax.group">
             <field name="name">TAX 5%</field>
         </record>
-        <record id="tax_group_7" model="account.tax.group">
-            <field name="name">TAX 7%</field>
-        </record>
         <record id="tax_group_vat_7" model="account.tax.group">
             <field name="name">VAT 7%</field>
```

Comment:
Should not this be TAX 7% instead of VAT 7% then?

Reply:
@jco-odoo  Done